algorithms/symmetry/cosym/target.py

```diff
@@ -84,15 +85,15 @@ def __init__(
                 last_id = lid
                 self._lattices.append(n)
 
-        self._sym_ops = OrderedSet(["x,y,z"])
+        self.sym_ops = OrderedSet(["x,y,z"])
```

Comment:
 Curious why you moved this from "_private"

Reply:
Because previously it was unnecessarily hidden behind a `self.get_sym_ops()` method (see 43f44952d78283350f47f3dc552ddc896001e470).

Reply:
I couldn't see the removal of the coresponding `get_sym_ops()` call in the commit -> wondering. Not a problem, just a question.

Reply:
Ah, because _deprecated_ and warning added -> gotcha.